package-lock.json

```diff
@@ -40689,6 +40703,13 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=",
       "dev": true
     },
+    "typescript": {
```

Comment:
is this a major change? in other words, does this imply other typescript use can start to proliferate around the project?

Reply:
no. it was pulled in by some dependency (wdio I think, didn't check too hard)

IMO it's best to ignore autogenerated files in reviews, if there's a problem, it's really about the config that generates it (package.json in this case), that's what should get attention.